scripts/aznfs_install.sh

```diff
@@ -4,29 +4,52 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
+APPNAME="aznfs"
+OPTDIR="/opt/microsoft/${APPNAME}"
+OPTDIRDATA="${OPTDIR}/data"
+LOGFILE="${OPTDIRDATA}/${APPNAME}.log"
+
 RELEASE_NUMBER=x.y.z
+REPO_OWNER="Azure"
+REPO_NAME="AZNFS-mount"
 AZNFS_RELEASE="aznfs-${RELEASE_NUMBER}-1"
 AZNFS_RELEASE_SUSE="aznfs_sles-${RELEASE_NUMBER}-1"
+SERVICE_NAME="manual-update"
```

Comment:
nit, instead of SERVICE_NAME, better name would be RUN_MODE
so users calls this script with a given run mode and it acts accordingly.